src/getHumanAge.test.js

```diff
@@ -1,10 +1,45 @@
 'use strict';
 
-describe('getHumanAge', () => {
+describe(`getHumanAge`, () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
+  test(`should be declared`, () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test(`should return 0's if both inputs <15`, () => {
+    expect(getHumanAge(14, 14))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 1 if inputs are 15-23`, () => {
```

Comment:
I would add separate checks for 15, 23 & 24

Reply:
added